kumascript/macros/EmbedLiveSample.ejs

```diff
@@ -52,8 +52,8 @@ var MIN_HEIGHT = 60;
 // Note that if the 'height' is a string like '100%' it won't be a
 // problem because `parseInt('100%') := 100` which is more than `MIN_HEIGHT`
 // assuming it's value is less than 100.
-if (height && parseInt(height) < MIN_HEIGHT) {
-  height = MIN_HEIGHT;
+if (height) {
+  height = Math.max(parseInt(height), MIN_HEIGHT);
 }
```

Comment:
[The failing test](https://github.com/mdn/yari/actions/runs/9600398330/job/26476498113?pr=11344#step:6:1616) makes me think that:
1. your change changes "100%" into "100",
2. you might want to do the same width `width` (with the caveat just mentioned),

```
    Expected: "<div class=\"code-example\"><div class=\"example-header\"></div><iframe class=\"sample-code-frame\" title=\"Examples sample\" id=\"frame_examples\" width=\"700px\" height=\"700px\" src=\"https://live.mdnplay.dev/en-US/docs/Web/CSS/flex-wrap/runner.html?id=examples\" sandbox=\"allow-same-origin allow-scripts\"></iframe></div>"
    Received: "<div class=\"code-example\"><div class=\"example-header\"></div><iframe class=\"sample-code-frame\" title=\"Examples sample\" id=\"frame_examples\" width=\"700px\" height=\"700\" src=\"https://live.mdnplay.dev/en-US/docs/Web/CSS/flex-wrap/runner.html?id=examples\" sandbox=\"allow-same-origin allow-scripts\"></iframe></div>"
```